### PR TITLE
Description changes

### PR DESCRIPTION
**Pull request by vibhabamba**
All details pages.
screen shot 2015-06-25 at 1 19 20 am

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -65,11 +65,19 @@
     </a>
     <img src="/images/tile1-details.gif" />
     <section class="headline">
-      Typography Update
+      Wikipedia Typography Update
     </section>
     <section class="details para">
       <p>
-        I lead a project to update the typography on Wikipedia Desktop &amp; mobile web. This included new font specifications, leading and sizing for headings &amp; body copy in 235 different languages including non latin scripts such as Hebrew, Arabic &amp; Navajo. This project was tested out on mobile for 6 months before being brought to desktop.
+        ROLE: I collaborated with another designer to initiate the Typography refresh in 2012 for mobile web & desktop. Prior to this typography update, we had more than 20 arbitrarily defined type sizes on desktop alone, low contrast for headings and incorrectly rendered glyphs. The type size was too small for readers, and the tight leading made long form reading quite difficult. We wanted our users to sense accuracy, reliability, and clarity from the core visual elements of Wikipedia, just like the actual content they were reading.
```
Comment (zahanm) [inline index.html]: you need to escape special characters.
specifically, `&` is `&amp;` in HTML

[Predefined entities in XML](https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references)

File: index.html [hunk 2]
```diff
@@ -89,8 +97,15 @@
     </section>
     <section class="details para">
       <p>
-        We created link previews to support quick investigation of links on Wikipedia.
-        This feature is currently testing on Greek and Catalan Wikipedia. It required 3 different layouts to support a cohesive image text layout. The key question was ‘How much content is needed for a user to get context?'
+        ROLE: I initiated and executed design for Link Preview cards as a 20% project. The readability of many first sentences is low on Wikipedia, due to writing style and usage of uncommon words. I believed that link previews could be  a cross platform component which combined the right amount of content and imagery to help a user learn quickly. Further, these could enable sharing of Wikipedia content on other networks.
+      </p>
+
```
Comment (zahanm) [inline index.html]: take out these blank newlines please.

File: index.html (hunk 1 above)
Comment (vibhabamba) [inline index.html]: Fixed

File: index.html (hunk 2 above)
Comment (vibhabamba) [inline index.html]: fixed.

**Comment by zahanm** [comment]
also, what about the styling of those capitalized headers.
want to make them bold instead? I don't think the capitalization looks that great.

**Comment by zahanm** [comment]
Looking better after code review.

**Pull request merged**